hack/.kindUtils

```diff
@@ -1,13 +1,35 @@
 # shellcheck shell=bash
 
+kindGenExternalKubeconfig() {
+  # Generate a kubeconfig that uses the docker bridge network IP address of the cluster
+  # This is required for using the subctl cmd (for submariner)
+  local master_ip
+  mkdir -p ./tmp/kubeconfigs/external/
+  EXTERNAL_KUBECONFIG=./tmp/kubeconfigs/external/${cluster}.kubeconfig
+  cp ./tmp/kubeconfigs/${cluster}.kubeconfig ${EXTERNAL_KUBECONFIG}
+  master_ip=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' "${cluster}-control-plane" | head -n 1)
+  yq -i ".clusters[0].cluster.server = \"https://${master_ip}:6443\"" "${EXTERNAL_KUBECONFIG}"
+  yq -i "(.. | select(. == \"kind-${cluster}\")) = \"${cluster}\"" "${EXTERNAL_KUBECONFIG}"
```

Comment:
This has added a dependency on yq, if it's not installed, you get:

```
Set kubectl context to "kind-mgc-control-plane"
./hack/.kindUtils: line 11: yq: command not found
make: *** [Makefile:83: local-setup] Error 127
```

We should either add it to .bin, or call it out somewhere how to install it.